### ISSUE TITLE
Why does `get_values` exist?

### ISSUE THREAD

**Issue by alifeee** [burnash/gspread#1285]
`gspread.Worksheet` offers two ways of getting a range of cells: [`Worksheet.get`] and [`Worksheet.get_values`].

Why?

...

They return almost the same thing? [`Worksheet.get`] returns a [`ValueRange`], which inherits from `list`. [`Worksheet.get_values`] returns a list. `get_values` pads the lists so that they are square, whereas `get` does not.

```python
v_get_values = worksheet.get_values("A3:C6")
v_get = worksheet.get("A3:C6")

print("v_get_values: ", v_get_values)
print("v_get: ", v_get)
```

![Screenshot of spreadsheet](https://github.com/burnash/gspread/assets/13833017/7ef2cce3-9f96-488c-9c95-1f4124b15ce1)

```bash
$ py .\get_vs_get_values.py
v_get_values:  [['2', 'b', '2'], ['3', 'c', ''], ['4', 'd', '']]
v_get:  [['2', 'b', '2'], ['3', 'c'], ['4', 'd']]
```

[`Worksheet.get`]: https://github.com/burnash/gspread/blob/9cc862bc24f61de763944c59b2e1d270ef3b48cf/gspread/worksheet.py#L779-L870
[`Worksheet.get_values`]: https://github.com/burnash/gspread/blob/9cc862bc24f61de763944c59b2e1d270ef3b48cf/gspread/worksheet.py#L364-L477
[`ValueRange`]: https://github.com/burnash/gspread/blob/9cc862bc24f61de763944c59b2e1d270ef3b48cf/gspread/worksheet.py#L41

Why is one required over another? I propose that `get` should also merge the values, and then `get_values` is not required (can just call `get`)

**Comment by alifeee** [comment]
This is when `get_values` was added: https://github.com/burnash/gspread/commit/ac2c3982ccb3582a1cb79f35328027c7d628b58a

**Comment by alifeee** [comment]
closed by #1296
